app/views/shared/_event_logs_table.html.erb

```diff
@@ -1,18 +1,18 @@
-<% if logs.any? %>
+<% if @logs.any? %>
```

Comment:
Can we use the local variable rather than the instance (`@`) for both `user` and `logs` here if we're passing them in in `app/views/account/activities/show.html.erb` or is there a reason we've gone for the instance var here?

Reply:
There we go, sorry, sloppy git fu